target_chains/starknet/tools/test_vaas/src/bin/generate_wormhole_vaas.rs

```diff
@@ -77,7 +77,7 @@ fn main() {
                 consistency_level: 6,
                 payload: PayloadKind::Binary(
                     GuardianSetUpgrade {
-                        chain_id: 1,
+                        chain_id: 60051,
```

Comment:
Normally chainId of 0 is used here to mention all target chains.

Reply:
Let's double check that wormhole governance also supports 0.

Reply:
Changed to 0 for this one. Note that we also use mainnet VAAs in tests and those also have chain_id == 0.